acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/models/PublicationBuilder.java

```diff
@@ -62,20 +63,24 @@ public PublicationBuilder withAttachments(final AttachmentBuilder... attachmentB
         return cloneAndAmend(builder -> builder.attachmentBuilders = asList(attachmentBuilders));
     }
 
-    public PublicationBuilder withStatus(final PublicationStatus status) {
+    public PublicationBuilder inStatus(final PublicationStatus status) {
```

Comment:
I had in mind "inState", which I think would sound clearer. And maybe `PublicationStatus` should be `PublicationState` ideally?

Reply:
Class and properties renamed.